content/katas/fibonacci sequence/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Fibonacci sequence
+pre: "<b>1. </b>"
+weight: 5
+---
+
+
+ **What is fibonacci sequence?**
+
+> Fibonacci Sequence: is the series of numbers where the next number is found by adding up the n numbers before it.
+> e.g. in this case n = 2:
+> sequence: 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, ...
+
+**1.** 
+write a function that takes in a number argument [n] and returns fibonacci sequence of the given number [n], "series of values" 
+
+```js
+//javascript
+function fibonacci(144)
+
+Expected output:
+
+0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144
+```
```

Comment:
This example is incorrect, because of the improper use of the Fibonacci sequence, please refactor.

correct application:
fibonacci(5) => [0, 1, 1, 2, 3]
fibonacci(10) => [0, 1, 1, 2, 3, 5, 8, 13, 21, 34]